extensions/hibernate-reactive/deployment/src/test/java/io/quarkus/hibernate/reactive/SchemaUpdateTest.java

```diff
@@ -28,12 +27,15 @@
     @Inject
     Mutiny.SessionFactory sessionFactory;
 
+    @RunOnVertxContext
```

Comment:
That's not enough, what kind of context is it running on?
Event loop context, worker context, duplicated context? 

The default should be a duplicated context, but we may need parameters to create the right type of context.

Reply:
Right - likek I suggested I believe we should have some attributes on the annotation to make things explicit and controllable.

But I think "duplicated" could be a reasonable default.

https://github.com/quarkusio/quarkus/compare/main...Sanne:CheckContext?expand=1#diff-b4dcc8850c70f1ef3ed1c7c55fd3cd5b12db20e8b41f3cf8904e0d6ef3536333L77-R86

Reply:
Cool, yes thanks a lot to implement the testing tool bits. I was trying yesterday but honestly got lost - not familiar with it at all so I appreciate you doing that part.

Reply:
Yeah, it's too easy to get lost with all the necessary classloader hacks...